recipes/gmp/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "6.1.0" %}
+
+package:
+  name: gmp
+  version: {{ version }}
+
+source:
+  fn: gmp-{{ version }}.tar.bz2
+  url: ftp://ftp.gnu.org/gnu/gmp/gmp-{{ version }}.tar.bz2
+  md5: 86ee6e54ebfc4a90b643a65e402c4048
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Yes, GMP cannot be built in MSVC. Hence for SymEngine I use GMP for Linux and OS X and MPIR for Windows.
Here is a PR for conda-recipes that never got merged. https://github.com/conda/conda-recipes/pull/145